cockatrice/src/dlg_edit_user.cpp

```diff
@@ -34,8 +34,8 @@ DlgEditUser::DlgEditUser(QWidget *parent, QString email, int gender, QString cou
     int i = 1;
     foreach(QString c, countries)
     {
-        countryEdit->addItem(QPixmap(":/resources/countries/" + c + ".svg"), c);
-        if(c == country)
+        countryEdit->addItem(QPixmap(":/resources/countries/" + c.toLower() + ".svg"), c);
```

Comment:
I don't see the point in calling .toLower() here, since the values of ```c``` comes from ```SettingsCache::getCountries()```, that returns hardcoded lowercase values.